### PR TITLE
Update of the basics of DML article; new score for PLIV; adaption due to changed API of DoubleMLPLR & DoubleMLPLIV

### PR DESCRIPTION
**Pull request by MalteKurz**
This PR contains:
* A major update of the basics of DML article. The graphics are now produced for the standardized estimates and compared to a normal density. Results for both the partialling out score and the IV-type score are shown. Additionally, the graphics have been prettified.
* Documentation for the newly added score `IV-type` for the PLIV model (see Python: https://github.com/DoubleML/doubleml-for-py/pull/151 and R: https://github.com/DoubleML/doubleml-for-r/pull/161)
* All articles, user guide etc have been updated in accordance with the changed API of DoubleMLPLR & DoubleMLPLIV (see Python: https://github.com/DoubleML/doubleml-for-py/pull/151 and R: https://github.com/DoubleML/doubleml-for-r/pull/161)

### CODE REVIEW

**Comment by MalteKurz** [comment]
* Build with released packages is expected to fail (updates in the pkg code needs to be released first)
* Build with dev packages is also expected to fail until https://github.com/DoubleML/doubleml-for-py/pull/151 and https://github.com/DoubleML/doubleml-for-r/pull/161 are merged
* However, with packages from branches the build can be triggered successfully, see https://github.com/DoubleML/doubleml-docs/actions/runs/2358365697

File: doc/guide/basics.rst [hunk 1]
```diff
@@ -233,57 +181,76 @@ other half of observations indexed with :math:`i \in I`
...
-                        xlim(c(0.08, 0.75)) + xlab("") + ylab("") + theme_minimal()
+        g_nonorth = ggplot(data.frame(theta_rescaled=(theta_nonorth - alpha)/se_nonorth)) +
+                        geom_histogram(aes(y=..density.., x=theta_rescaled, colour = "Non-orthogonal ML", fill="Non-orthogonal ML"),
+                                       bins = 30, alpha = 0.3) +
+                        geom_vline(aes(xintercept = 0), col = "black") +
+                        suppressWarnings(geom_function(fun = dnorm, aes(colour = "N(0, 1)", fill="N(0, 1)"))) +
+                        scale_color_manual(name='',
+                             breaks=c("Non-orthogonal ML", "N(0, 1)"),
+                             values=c("Non-orthogonal ML"="dark blue", "N(0, 1)"='black')) +
+                        scale_fill_manual(name='',,
```
Comment (PhilippBach) [inline doc/guide/basics.rst:234]:
```suggestion
                        scale_fill_manual(name='',
```

File: doc/guide/basics.rst [hunk 2]
```diff
@@ -340,43 +320,58 @@ orthogonalized regressor :math:`V = D - m(X)`. We then use the final estimate
...
-                    xlim(c(0.08, 0.75)) + xlab("") + ylab("") + theme_minimal()
+        g_nosplit = ggplot(data.frame(theta_rescaled=(theta_orth_nosplit - alpha)/se_orth_nosplit), aes(x = theta_rescaled)) +
+                        geom_histogram(aes(y=..density.., x=theta_rescaled, colour = "Double ML (no sample splitting)", fill="Double ML (no sample splitting)"),
+                                       bins = 30, alpha = 0.3) +
+                        geom_vline(aes(xintercept = 0), col = "black") +
+                        suppressWarnings(geom_function(fun = dnorm, aes(colour = "N(0, 1)", fill="N(0, 1)"))) +
+                        scale_color_manual(name='',
+                             breaks=c("Double ML (no sample splitting)", "N(0, 1)"),
+                             values=c("Double ML (no sample splitting)"="dark orange", "N(0, 1)"='black')) +
+                        scale_fill_manual(name='',,
```
Comment (PhilippBach) [inline doc/guide/basics.rst:358]: I think there is one `,` too much
```suggestion
                        scale_fill_manual(name='',
```

File: doc/guide/basics.rst [hunk 3]
```diff
@@ -416,30 +424,45 @@ Cross-fitting performs well empirically because the entire sample can be used fo
...
-                    xlim(c(0.08, 0.75)) + xlab("") + ylab("") + theme_minimal()
+        g_dml = ggplot(data.frame(theta_rescaled=(theta_dml - alpha)/se_dml), aes(x = theta_rescaled)) +
+                        geom_histogram(aes(y=..density.., x=theta_rescaled, colour = "Double ML with cross-fitting", fill="Double ML with cross-fitting"),
+                                       bins = 30, alpha = 0.3) +
+                        geom_vline(aes(xintercept = 0), col = "black") +
+                        suppressWarnings(geom_function(fun = dnorm, aes(colour = "N(0, 1)", fill="N(0, 1)"))) +
+                        scale_color_manual(name='',
+                             breaks=c("Double ML with cross-fitting", "N(0, 1)"),
+                             values=c("Double ML with cross-fitting"="dark green", "N(0, 1)"='black')) +
+                        scale_fill_manual(name='',,
```
Comment (PhilippBach) [inline doc/guide/basics.rst:462]: I think there is one `,` too much
```suggestion
                        scale_fill_manual(name='',
```

**Review by PhilippBach — COMMENTED** [review]
Thanks @MalteKurz for preparing this PR which includes a lot of changes regarding the (names) of the nuisance learners as well as the example in the basics chapter. I only have a  minor comment/suggestion for the plots in R there

**Pull request merged**